printer_mods/LoCoCNC/ssr_strap/readme.md

```diff
@@ -0,0 +1,5 @@
+SSR Strap for attaching the V0 SSR to a 1515 extrusion.
+
+
+![Image of Strain Relief 3](./ssr_strap.JPG)
```

Comment:
Please update the text from `Image of Strain Relief 3` to something like `SSR Mount`

Reply:
Done